### PR TITLE
Phantomjs

### PR DESCRIPTION
**Pull request by gilgamez**
See https://app.wercker.com/#build/574c469fae1a1b664805b923 for the effect. open the tab `Install dependencies`

### CODE REVIEW

**Comment by Voles** [comment]
Ready for review?

File: Dockerfile [hunk 1]
```diff
@@ -25,9 +25,15 @@ RUN \
     istanbul \
     knex \
     mocha \
-    phantomjs@"1.9.19" \
     protractor
 
+ENV PHANTOMJS_NAME phantomjs-1.9.8-linux-x86_64
+
+RUN \
+  curl --silent --show-error --location https://cnpmjs.org/downloads/$PHANTOMJS_NAME.tar.bz2 | \
```
Comment (timenolthof) [inline Dockerfile]: Can't we just check the file in to this repository for even less dependencies?

**Comment by timenolthof** [comment]
RSC

File: Dockerfile (hunk 1 above)
Comment (gilgamez) [inline Dockerfile]: That is what eventually happens by building the docker image

**Comment by jvandereijk** [comment]
LGTM

File: Dockerfile (hunk 1 above)
Comment (timenolthof) [inline Dockerfile]: Yes and that happens every time the image is built. Why is it better to just add the filename here instead of the file?

Comment (gilgamez) [inline Dockerfile]: And when we decide to bump the phantom js version we only have to change this var and not upload it to github again.
It's not nice to store binaries in a version control system.

**Comment by robgietema** [comment]
LGTM

**Pull request merged**